### PR TITLE
[0.13.x] Update SPEC file to pick the latest go version

### PR DESCRIPTION
**Pull request by navidshaikh**

### CODE REVIEW

File: openshift-serverless-clients.spec [hunk 1]
```diff
@@ -4,7 +4,7 @@
 %global product_name OpenShift Serverless
 %global golang_version 1.13
```
Comment (warrenvw) [inline openshift-serverless-clients.spec:5]: If specific version of go is required here, then you can use `golang_version 1.13.15`.

**Comment by navidshaikh** [comment]
/hold

openshift-serverless-clients RPM is not multi-stream, we'll only have one "latest" version of RPM. :-/

**Pull request closed without merging**